code/src/components/Form.js

```diff
@@ -0,0 +1,296 @@
+									alt={alt}
+									onChange={event => setBreakupPerson(event.target.value)}
+									checked={breakupPerson === alt}
+								/>
+							</div>
+						)}
+					</div>
+					<div>
+						<h3 className='error'>{error}</h3>
+						<Button
+							key={'button-back'}
+							onClick={backPage}
+							text='Back'
+						/>
+						<Button
+							key={'button-next'}
+							onClick={() => nextPage(breakupPerson)}
+							text='Next'
+						/>
+					</div>
+				</div>
+			)}
+
+			{page === 3 && (
+				<div className='current-page text-input-container'>
+					<Input 
+						labelClassName={'text-label'}
+						label={<h2>{questions.breakupReasonQuestion}</h2>}
+						onChange={handleBreakupReason}
+						value={breakupReason}
+						placeholder={shipDuration + ' ago you insinuated that...'}
+					/>
+					<div>
+						<h3 className='error'>{error}</h3>
+						<Button
+							key={'button-back'}
+							onClick={backPage}
+							text='Back'
+						/>
+						<Button
+							key={'button-next'}
+							onClick={() => nextPage(breakupReason)}
+							text='Next'
+						/>
+					</div>
+				</div>
+			)}
+			{page === 4 && (
+				<div className='current-page question'>
+					<h2>{questions.traitsQuestion}</h2>
+					<div className='traits-container'>
+						{questions.traits.map((trait) => 
+							<Input 
+								labelKey={trait}
+								labelClassName={'trait'}
+								label={trait}
+								onChange={() => handleTraitsCheckboxesChange(trait)}
+								type={'checkbox'}
+							/>
+						)}
+					</div>
+					<div>
+						<h3 className='error'>{error}</h3>
+						<Button
+							key={'button-back'}
+							onClick={backPage}
+							text='Back'
+						/>
+						<Button
+							key={'button-next'}
+							onClick={() => nextPage(traitsCheckboxes)}
+							text='Next'
+						/>
+					</div>
+				</div>
+			)}
+			{page === 5 && (
+				<div className='current-page question'>
+					<div>
+						<h2>{questions.backTogetherQuestion}</h2>
+						{questions.backTogetherAlts.map((alt) =>
+							<div key={alt}>
+								<RadioButton
+									alt={alt}
+									onChange={handleGetBackTogether}
+									checked={getBackTogether === alt}
+								/>
+							</div>
+						)}
+					</div>
+					{(getBackTogether === "Never in a million years!" || getBackTogether === "Nah.") && (
+						<div>
+							<h2>{questions.dateFriendQuestion}</h2>
+							{questions.backTogetherAlts.map((alt, index) =>
+								<div key={alt+index}>
+									<RadioButton
+										alt={alt}
+										onChange={handleDateFriend}
+										checked={dateFriend === alt}
+									/>
+								</div>
+							)}
+						</div>
+					)}
+					{(dateFriend === "Absolutely!" || dateFriend === "Maybe?") && (
+						<div className='text-input-container'>
+							<Input 
+								label={<h2>{questions.friendNameQuestion}</h2>}
+								onChange={handleFriendContactDetails}
+								value={friendContactDetails}
+								placeholder={'Name, number'}
+							/>
+						</div>
+					)}

```

Comment:
So cool that you put a conditional in here depending on what people answered... Love love love!!!